packages/connection/src/class-client.php

```diff
@@ -445,18 +441,13 @@ public static function _stringify_data( $data ) { // phpcs:ignore PSR2.Methods.M
 	/**
 	 * Gets protocol string.
 	 *
-	 * @return string `https` (if possible), else `http`.
+	 * @return string Always 'https'.
+	 *
+	 * @deprecated 9.1.0 WP.com API no longer supports requests using `http://`.
 	 */
 	public static function protocol() {
-		/**
-		 * Determines whether Jetpack can send outbound https requests to the WPCOM api.
-		 *
-		 * @since 3.6.0
-		 *
-		 * @param bool $proto Defaults to true.
-		 */
-		$https = apply_filters( 'jetpack_can_make_outbound_https', true );
```

Comment:
Can you add this to the `$filter_deprecated_list` array in `class.jetpack.php`?

Reply:
Done 👍